src/Telegram.php

```diff
@@ -396,32 +389,31 @@ public function handle()
      *
      * @return \Longman\TelegramBot\Telegram
      */
-
-    public function processUpdate(update $update)
+    public function processUpdate(Update $update)
     {
         $update_type = $update->getUpdateType();
         if ($update_type == 'message') {
             //Load admin Commands
             if ($this->admin_enabled) {
                 $message = $update->getMessage();
-    
+
                 //Admin command avaiable in any chats
                 //$from = $message->getFrom();
                 //$user_id = $from->getId();
-    
+
                 //Admin command avaiable only in single chat with the bot
                 $chat = $message->getChat();
                 $user_id = $chat->getId();
-    
+
                 if (in_array($user_id, $this->admins_list)) {
-                    $this->addCommandsPath(BASE_PATH.'/Admin');
+                    $this->addCommandsPath(BASE_PATH . '/Admin');
                 }
             }
-    
+
             // check type
             $message = $update->getMessage();
             $type = $message->getType();
-    
+
             switch ($type) {
```

Comment:
All of these command types, apart from the "Generic" ones, follow the same conversion rules.
- Remove "_"
- Capitalise first character
For that reason, this part can be shortened by quite a bit.

At the same time I can see that having each command on a separate `case`, additional code could be added on a case by case basis. Although this could also be done for individual cases specifically: `if..then..`

Shall I attempt to shorten this function or do you want to leave it as it?